cheroot/test/test_ssl.py

```diff
@@ -137,6 +148,14 @@ def make_tls_http_server(bind_addr, ssl_adapter, request):
     return httpserver
 
 
+@pytest.fixture
+def private_key_password():
+    """Provide random 10 character password for private key."""
+    return ''.join(
+        secrets.choice(string.ascii_letters + string.digits) for _ in range(10)
+    )
```

Comment:
Let's just keep it simple
```suggestion
    """Return a password for private key."""
    return 'криївка'
```